js/popup.js

```diff
@@ -1,486 +1,462 @@
+    document.getElementById('logo-correlate').onclick = btnQuantiModoLogoCorrelateClick;
+    document.getElementById('logo-correlate-1').onclick = btnQuantiModoLogoCorrelateClick;
+    document.getElementById('logo-correlate-2').onclick = btnQuantiModoLogoCorrelateClick;
+
+    //var optionsUrl = chrome.extension.getURL("src/options/options.html");
+    //var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
 
 
-		
 };
 
 /**
  * Record a measurement is clicked
  */
-var onQmRcdMstButtonClicked = function()
-{
-	var name = $("#addmeasurement-variable-name").val();
+var onQmRcdMstButtonClicked = function () {
+    var name = $("#addmeasurement-variable-name").val();
+
+    if (name == '') {
 
-	if (name == '') {
-		alert("Please enter the variable name."); return;
-	}
+        $('.validation-holder span').text('Please enter the variable name');
+        $('#addmeasurement-variable-name').addClass('error');
 
-	var n_value = getVariableWithName(name);
+        return;
+    } else {
 
-	if ( n_value == null )
-	{
-		$("#record_a_measurement_block").hide();
-		$("#add_record_a_measurement_block").show();
-		$("#add-addmeasurement-variable-name").val(name);
+        $('.validation-holder').text('');
+        $('#addmeasurement-variable-name').removeClass('error');
 
-	} 
-	else 
-	{
-		$("#record_a_measurement_block").hide();
-		$("#edt_record_a_measurement_block").show();
-		$("#edt-addmeasurement-variable-name").val(name);
+    }
 
-	}
+    var n_value = getVariableWithName(name);
 
-}
+    if (n_value == null) {
+        $("#record_a_measurement_block").hide();
+        $("#add_record_a_measurement_block").show();
+        $("#add-addmeasurement-variable-name").val(name);
 
+    }
+    else {
+        $("#record_a_measurement_block").hide();
+        $("#edt_record_a_measurement_block").show();
+        $("#edt-addmeasurement-variable-name").val(name);
+
+    }
+
+}
 
 
 // registration button clicked
-var btnQuantiModoRegisterClick = function()
-{
-	var optionsUrl = chrome.extension.getURL("options/options.html"); 
-	//var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
+var btnQuantiModoRegisterClick = function () {
+    var optionsUrl = chrome.extension.getURL("options/options.html");
+    //var content = '<a href="' + optionsUrl + '" target="_blank">Options</a>';
 
-	chrome.tabs.create({ url: optionsUrl });
+    chrome.tabs.create({url: optionsUrl});
 };
 
 
 // registration button clicked
-var btnQuantiModoLogoCorrelateClick = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/correlate/" });
+var btnQuantiModoLogoCorrelateClick = function () {
+    chrome.tabs.create({url: "https://app.quantimo.do/correlate/"});
 };
 
 
 // registration button clicked
-var onRegisterAClicked = function()
-{
-	chrome.tabs.create({ url: "https://app.quantimo.do/citizen-science/register/" });
+var onRegisterAClicked = function () {

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['onRegisterAClicked' is defined but never used.](https://www.codacy.com/app/m_3/QuantiModo-Chrome-Extension/file/832637814/issues/source?bid=2779649&fileBranchId=2790922#l125)